submissions/marwan_alrayes/assignment8.js

```diff
@@ -0,0 +1,42 @@
+// task1
+const editor = {
+    title: "my first blog",
+    getUpperTitle() {
+        return this.title.toUpperCase();
+    },
+};
+// the edit is
+const getTitle = editor.getUpperTitle.bind(editor);
+console.log(getTitle());
+
+//-----------------------------------------------------------
+
+// task2
+const formHandler = {
+    value: "initial",
+    onChange(newValue) {
+        this.value = newValue;
+    },
+};
+function simulateInputChange(callback) {
+    callback("updated");
+}
+// the edit is
+const boundOnChange = formHandler.onChange.bind(formHandler);
+simulateInputChange(boundOnChange);
+console.log(formHandler.value);
+
```

Comment:
Please think how we can solve it using arrow function and explain both of ur solution 
and finally please predict the value in line 27